src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -19,8 +19,9 @@
           Inputs="@(WebpackInputs)"
           Outputs="@(WebPackOutputs)">
     <RemoveDir Directories="dist" />
-    <Yarn Command="run build:debug" />
-    <Yarn Command="run build:production" />
+
+    <Exec Command="yarn run build:debug" />
```

Comment:
This is going to require yarn be pre-installed globally...which it isn't in most of our dev environments. Suggest doing this instead:
```
<Yarn Command="run build:debug" StandardOutputImportance="High" />
```

Reply:
Okies. Let me get the diagnose the log from the build and I'll change the codez.